ev3dev2/motor.py

```diff
@@ -1781,13 +1863,47 @@ def on_for_degrees(self, left_speed, right_speed, degrees, brake=True, block=Tru
         self.right_motor._set_rel_position_degrees_and_speed_sp(right_degrees, right_speed_native_units)
         self.right_motor._set_brake(brake)
 
+        left_motor_runtime_eta = float(left_degrees / left_speed_native_units)
```

Comment:
`(degrees) / (native units per second) != time`; what if native units aren't degrees?

Reply:
I am going to chop the ETA debugs it feels like overkill.